Source/Layer.js

```diff
@@ -74,60 +192,115 @@ export class Layer extends WrappedObject {
       return frame
     }
 
+    /**
+        Returns a newly created shape, which has been added to this layer,
+        and sets it up using the supplied properties.
+
+        @param properties {dictionary} Properties to apply to the shape.
```

Comment:
Yeah, thanks - I thought that there were probably still a few of those lurking...